www/src/components/Menu.jsx

```diff
@@ -96,6 +98,11 @@ const MenuComponentListCategory = ({ children, title }) => (
   </div>
 );
 
+const handleButtonClick = () => {
```

Comment:
nit: maybe we should be a bit more specific with the naming here? e.g. `handlePlaygroundClick` so it's obvious from the name that it's a handler for playground button and not some other button on the page

Reply:
Renamed